website/source/guides/ui.html.md

```diff
@@ -1,31 +1,33 @@
 ---
 layout: "guides"
-page_title: "Nomad UI"
+page_title: "Web UI"
 sidebar_current: "guides-ui"
 description: |-
-  The Nomad UI is a great companion for both operators and developers.
+  The Nomad Web UI is a great companion for both operators and developers.
   It is an easy to use way to inspect jobs, allocations, and nodes.
 ---
 
-# Nomad UI
+# Web UI
 
-The Nomad UI offers an easy to use web experience for inspecting a Nomad cluster.
+The Nomad Web UI offers an easy to use web experience for inspecting a Nomad cluster.
 Jobs, Deployments, Task Groups, Allocations, Clients, and Servers can all be
-monitored from the Nomad UI. The UI also supports the use of ACL tokens for
-clusters that are using the Nomad Access Control List system.
+monitored from the Web UI. The Web UI also supports the use of ACL tokens for
+clusters that are using the [ACL system](/guides/acl.html).
 
-~> **Beta Feature!** This page covers the Nomad UI, a feature of v0.7 which is still in beta.
+~> **Beta Feature!** This page covers the Web UI, a feature of v0.7 which is still in beta.
 
-## Accessing the UI
+## Accessing the Web UI
 
-The Nomad UI is served on the same address and port as the HTTP API. It is namespaced
+The Web UI is served on the same address and port as the HTTP API. It is namespaced
 under `/ui`, but visiting the root of the Nomad server in your browser will redirect you
-to the UI. If you are unsure what port the Nomad HTTP API is running under, try the default
+to the Web UI. If you are unsure what port the Nomad HTTP API is running under, try the default
 port, `4646`.
 
+For a quick test drive, see our online Web UI demo at [demo.nomadproject.io](https://demo.nomadproject.io). 
```

Comment:
Maybe put this in a call out. People are going to love it!

Reply:
Done.